enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.unit.spec.tsx

```diff
@@ -36,17 +39,17 @@ import { createMockState } from "metabase-types/store/mocks";
 
 const TEST_USER = createMockUser();
 
-const TestComponent = ({ config }: { config: SDKConfig }) => {
+const TestComponent = ({ config }: { config: MetabaseAuthConfig }) => {
```

Comment:
nit: I think we should also rename this

```suggestion
const TestComponent = ({ authConfig }: { authConfig: MetabaseAuthConfig }) => {
```